src/main/java/seedu/contax/ui/HelpWindow.java

```diff
@@ -66,37 +90,193 @@ public void show() {
         logger.fine("Showing help page about the application.");
         getRoot().show();
         getRoot().centerOnScreen();
+
     }
 
     /**
-     * Returns true if the help window is currently being shown.
+     * Returns true if the help window is currently being shown
      */
     public boolean isShowing() {
         return getRoot().isShowing();
     }
 
     /**
-     * Hides the help window.
+     * Hides the help window
      */
     public void hide() {
         getRoot().hide();
     }
 
     /**
-     * Focuses on the help window.
+     * Focuses on the help window
      */
     public void focus() {
         getRoot().requestFocus();
     }
 
     /**
-     * Copies the URL to the user guide to the clipboard.
+     * Initializes the HelpWindow by populating it with data
      */
-    @FXML
-    private void copyUrl() {
-        final Clipboard clipboard = Clipboard.getSystemClipboard();
-        final ClipboardContent url = new ClipboardContent();
-        url.putString(USERGUIDE_URL);
-        clipboard.setContent(url);
+    private void init() {
+
+        // set up table headers
+        insertHeaderRow("Action", "Format", "Example", generalPage);
+        insertHeaderRow("Action", "Format", "Example", personsPage);
+        insertHeaderRow("Action", "Format", "Example", appointmentsPage);
+        insertHeaderRow("Action", "Format", "Example", tagsPage);
+
+        // populate general page
+        rowCounter = 1;
```

Comment:
Would it better if these `insertRow()` statements can be abstracted into a general function which fills in each row entry of the pane?
```suggestion
        fillRows(GridPane page, String ...columns)
        // or fillRows(GridPane page, String[] ...table)
```

Reply:
Thanks for the suggestion! Will consider it